.travis.yml

```diff
@@ -33,22 +33,28 @@ before_install:
 
 script:
   - |
+    excludes=()
+    if [[ "${TRAVIS_OS_NAME}" == "osx" ]]; then
+      # https://github.com/bazelbuild/rules_closure/issues/247
+      excludes+=('//closure/testing/test:noto_fonts_render_as_expected')
```

Comment:
I would welcome a PR commenting out that test rule, saying, "hey this sometimes works, and is an example of what you *could* do, depending on your environment." If you do this, please also revert the complexity that rule added to this file.